windows_docker_resources/Dockerfile

```diff
@@ -0,0 +1,46 @@
+# escape=`
+# This Dockerfile needs to be built from the parent directory (ros2/ci) so the build context
+# includes the python scripts
+
+# To find this value run in powershell:
+# $(Get-ItemProperty 'HKLM:\SOFTWARE\Microsoft\Windows NT\CurrentVersion').ReleaseId
+ARG WINDOWS_RELEASE_ID=1909
+
+# Indicates that the windows image will be used as the base image. Must be same or older than host.
+# --isolation=hyperv is needed for both build/run if the image id is older than the host id
+# Use --isolation=process if you need to build in a mounted volume
+# FROM mcr.microsoft.com/windows:$WINDOWS_RELEASE_ID
+FROM mcr.microsoft.com/windows@sha256:27f1a2a1f2f784f3716c2400d7e1bf93074a75e15f2974861c00275f93487c88
+
+# Install cinc-solo, a compiled binary of chef-solo
+RUN powershell "iex  ((New-Object System.Net.WebClient).DownloadString('https://omnitruck.cinc.sh/install.ps1')); install -version 15.10.12"
+
+# Install Chocolatey by powershell script
+RUN powershell -noexit "Set-ExecutionPolicy Bypass -Scope Process -Force; iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'))"
+
+# choco installs
+RUN choco install -y git chefdk
```

Comment:
I expect the chefdk chocolatey package is installing the entire Chef Development Kit which is more than we need in these containers and contains components which are license encumbered.
I would prefer installing cinc via omnitruck unless there is some reason that's insufficient.

Reply:
Ah, are we using it to install berks?

Reply:
Yah, it's  for berks